base/util.go

```diff
@@ -19,11 +19,11 @@ import (
 
 // MajorMinor is the minor version of Iter8
 // set this manually whenever the major or minor version changes
-var MajorMinor = "v0.17"
+var MajorMinor = "v0.18"
 
 // Version is the semantic version of Iter8 (with the `v` prefix)
 // Version is intended to be set using LDFLAGS at build time
-var Version = "v0.17.0"
+var Version = "v0.18.1"
```

Comment:
Does this need to be `.1`? We've always left the patch version as `.0`.

Reply:
done